<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TulipWelcomePageData</class>
 <widget class="QWidget" name="TulipWelcomePageData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>9</number>
     </property>
     <property name="margin">
      <number>9</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;

&lt;body&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/tulip/gui/ui/list_bullet_arrow.png&quot; /&gt;&lt;span style=&quot;color:#626262; font-size:large;&quot;&gt;   Manage your projects&lt;/span&gt;&lt;/p&gt;

&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:medium;&quot;&gt; &lt;/p&gt;

&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot;color:#000000;&quot;&gt;Start a new project with Tulip using one of the installed perspectives&lt;/span&gt;&lt;/p&gt;
&lt;/body&gt;

&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="perspectivesScrollArea">
       <property name="styleSheet">
        <string notr="true">#perspectivesFrame {
  background-color: rgb(255, 255, 255);
  border-left: 1px solid &quot;#C9C9C9&quot;;
  border-right: 1px solid &quot;#C9C9C9&quot;;
  border-top: 1px solid &quot;#C9C9C9&quot;;
  border-bottom: 1px solid &quot;#C9C9C9&quot;;
}

QToolButton {
	border-image:none;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0.0, y2: 1.0,
	stop: 0 #ffffff,
	stop: 0.4 #eeeeee,
	stop: 0.401 #e2e2e2,
	stop: 1 #dddddd);
	color: black;
	border-top: 1px solid #606060;
	border-bottom: 1px solid #404040;
}

QToolButton:hover {
	border-image:none;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0.0, y2: 1.0,
	stop: 0 #909090,
	stop: 0.4 #808080,
	stop: 0.401 #707070,
	stop: 1 #555555);
}

QToolButton:checked, QToolButton:checked:pressed {
	border-image:none;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0.0, y2: 1.0,
	stop: 0 #ffffff,
	stop: 0.4 #eeeeee,
	stop: 0.401 #e2e2e2,
	stop: 1 #dddddd);
	color: black;
	border-top: 1px solid #606060;
	border-bottom: 1px solid #404040;
}

QToolButton:pressed {
	border-image:none;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0.0, y2: 1.0,
	stop: 0 #383838,
	stop: 0.2 #404040,
	stop: 0.201 #484848,
	stop: 1 #505050);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="perspectivesFrame">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>688</width>
          <height>601</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3"/>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="openProjectButton">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>36</height>
          </size>
         </property>
         <property name="text">
          <string>Open project</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+O</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>&lt;body&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/tulip/gui/ui/list_bullet_arrow.png&quot; /&gt;&lt;span style=&quot;color:#626262; font-size:large;&quot;&gt;    Recent projects&lt;/span&gt;&lt;/p&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="recentDocumentsLabel">
       <property name="text">
        <string>There is no recent document to display</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="sideFrame">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#sideFrame { 
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(245, 245, 245, 255), stop:1 rgba(225,225,225, 255)); 
border-left: 1px solid &quot;#C9C9C9&quot;;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;

&lt;body&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/tulip/gui/ui/list_bullet_arrow.png&quot; /&gt;&lt;span style=&quot;color:#626262; font-size:large;&quot;&gt;   Latest headlines&lt;/span&gt;&lt;/p&gt;

&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:medium;&quot;&gt; &lt;/p&gt;

&lt;/body&gt;

&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rssError">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;

&lt;body&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot;color:#dc4d4f;  font-size:medium;&quot;&gt;Unable to fetch latest headlines from Tulip website.&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot;color:#dc4d4f;  font-size:medium;&quot;&gt;Please check your network.&lt;/span&gt;&lt;/p&gt;

&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="rssScroll">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>238</width>
           <height>320</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="websiteLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;

&lt;body&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;http://tulip.labri.fr&quot;&gt;&lt;span&gt;Full RSS&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;body&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/tulip/gui/ui/list_bullet_arrow.png&quot; /&gt;&lt;span style=&quot;color:#626262; font-size:large;&quot;&gt;    Did you know?&lt;/span&gt;&lt;/p&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="previousTipButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/tulip/app/icons/arrow-left.png</normaloff>:/tulip/app/icons/arrow-left.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextTipButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/tulip/app/icons/arrow-right.png</normaloff>:/tulip/app/icons/arrow-right.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>recentDocumentsLabel</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>TulipWelcomePageData</receiver>
   <slot>recentFileLinkActivated(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>767</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>recentFileLinkActivated(QString)</slot>
 </slots>
</ui>
